src/components/Chat/RoomMenu.tsx

```diff
@@ -95,6 +99,7 @@ export const RoomButtons: React.FC<RoomMenuProps> = ({ onChange, onClose }) => {
     []
   );
 
+  const handleToggleClick = useCallback(() => toggleShowChat(), []);
```

Comment:
this shouldn't be defined here. the behavior for the button is defined by the value given to `onClose` which comes from here https://github.com/MemeLabs/strims/blob/main/src/components/Layout/Chat.tsx#L67

this should become something like 
```ts
const handleRoomMenuClose = useCallback(
  () => menuLocked ? toggleShowChat() : toggleMenuOpen(false), 
  [menuLocked],
);
```